### ISSUE TITLE
One-line summary

### ISSUE THREAD

**Issue by srmora** [ifrotz/iosfrotz#300]
The problem is when I link to my dropbox account I cannot access the saved file from another device. I can create a new save file but the other device on the other side does not see it

What steps will reproduce the problem?
1. I attempted to re-link my Dropbox account and then I went to dropbox to validate
2.I could see the initial folder from both devices to the dropbox
3.I even tried to uninstall and then reinstall the app and re-link again but still the same issue

What is the expected output or behavior?  What do you see instead? Are you expected to be able to access the same file from either my iPad or my iPhone of the same game but I could not

What version of Frotz are you using?
1.7.1
I believe I am using the latest version 1.81

What device model are you using (e.g., iPhone 6, iPad 4, iPod Touch 4th gen.)
iPhone 11 Pro and an iPad Mini 5

What version of iOS?  (6.1, 7.0, etc.)
13.1.2

Please provide any additional information below.
Aside from unable to access the saved file from either device everything works beautifully thank you

**Comment by spathiwa** [comment]
Odd, no one has every reported an issue like this with Dropbox.
Have you tried this with other games?  If it's happening on a non-built-in game, I have a suspicion that maybe there's a slight version or filename difference between the game you downloaded on your iPhone vs. your iPad, so the automatically generated folder containing the saves would be different.  Can you check your Dropbox account directly from a Mac/PC or from the DB App and see if that might be the case?

**Comment by srmora** [comment]
Thank you for the reply, somehow it fixed itself. I kept adding removing the app and re-linking the dropbox connection until I was able to access the saved game file from either device. I had recently moved to new devices the new iPhone Pro 11 and the iPad Mini five. So I went to try and use the sharing features it was the first adventure game which was a new one that was acting strange. I decided to try it with some of the other games I played and oddly enough they all worked fine so I figured it was just that game and I figured maybe something wrong with it. Anyway after trial and error of removing the game and testing trying to save suddenly the game file appeared and they were able to both access it. I did further testing and all seems to be all back to normal again. I’m sorry if I jumped the gun, but he had been functioning so perfectly for years and then suddenly The one particular adventure game I was playing it would not work. Anyway thanks again it’s a beautiful app I’ve been a long-time fan of interactive fiction. I’m even writing my own. Thanks again


--
Stephen Mora
courage.loyalty.self-discipline.simple living
